### ISSUE TITLE
"<<" Button to toggle participants is missing in mobile view mode

### ISSUE THREAD

**Issue by poVoq** [conversejs/converse.js#2270]
**Describe the bug**
In mobile view mode the "<<" button has disappeared in the recent master versions (compiled from source). Therefore once you close the participants list via the "x" button there is no way to restore it.

**Environment (please complete the following information):**
 - Desktop or Mobile? Mobile
 - Browser [e.g. Chrome, Firefox] Embedded chromium
 - Converse.js version [e.g. 4.2.0] 6.0.1dev / 7.0 (compiled from master 20201006)

**Comment by bernard-ng** [comment]
Hi @poVoq , can you provide a screenshot ?

**Comment by poVoq** [comment]
@bernard-ng Right now not as I am away from my dev PC, but it looks exactly the same as normal, just the button is missing.

**Comment by bernard-ng** [comment]
it seems to me that this is already fixed
HEAD : 646542602fbe65819a4e933744e28f1ae4f9b9a4

![WhatsApp Image 2020-10-21 at 16 45 25](https://user-images.githubusercontent.com/31113941/96736676-2080f100-13bd-11eb-9078-9c3362d3b0b6.jpeg)

**Comment by poVoq** [comment]
Hmm, maybe a browser specific thing? Can you try to replicate it in something Chromium based please?

**Comment by bernard-ng** [comment]
@poVoq 

![Screenshot_20201023-145526.png](https://user-images.githubusercontent.com/31113941/97006321-fd327f00-153f-11eb-82f9-bcb5cbb1879f.png)

**Comment by poVoq** [comment]
@bernard-ng  This is still happening here. After some debugging I found that it is related to setting:
`visible_toolbar_buttons: {emoji: false},`
If I remove this the toggle participants button shows up again.

Oh and I noticed that eye button for writing hidden messages (never noticed that one before, is it new?) also does not show up with the above setting.

IMHO hiding the on mobile view mode mostly broken emoji picker is quite important, as it is super slow, doesn't work well and is anyways unneeded as most OSK have emojis build in.
